src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/helpers/AdsVeles.java

```diff
@@ -0,0 +1,5 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.helpers;
+
+public class AdsVeles {
+  public static String TURN_ON_QUERY_PARAM = "InstantGlobals.wgAdDriverVelesBidderCountries=[XX]";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make TURN_ON_QUERY_PARAM a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this "public static TURN_ON_QUERY_PARAM" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1444)